internal/relationtuple/definitions.go

```diff
@@ -660,6 +662,10 @@ func NewManagerWrapper(_ *testing.T, reg ManagerProvider, options ...x.Paginatio
 	}
 }
 
+func (t *ManagerWrapper) Config() *config.Config {
```

Comment:
@zepatrik I realised that this was a lazy implementation because I wasn't sure how to separate ManagerWrapper from the EngineDependencies for check and expand in the tests. I think because there was a single dependency the tests have some coupling between EngineDependencies and ManagerProvider. This approach results in coupling of unrelated things. Manager and Config. Adding a Logger would exacerbate things. I'll push a commit that separates them in the tests and in production.